src/zenml/models/model_models.py

```diff
@@ -143,44 +153,55 @@ def pipeline_runs(self) -> Dict[str, PipelineRunResponseModel]:
             for name, pr in self.pipeline_run_ids.items()
         }
 
-    def get_model_object(self, name: str) -> ArtifactResponseModel:
-        """Get model object linked to this version.
+    def get_model_object(self, name: str) -> Dict[str, ArtifactResponseModel]:
```

Comment:
IMO think would be a more intuitive interface - same for the other two methods below

```suggestion
    def get_model_object(self, name: str, version:str) -> ArtifactResponseModel:
```